packages/server-wallet/src/protocols/__test__/close-channel.test.ts

```diff
@@ -1,70 +1,151 @@
-import {simpleEthAllocation, BN, State} from '@statechannels/wallet-core';
-
-import {protocol} from '../close-channel';
-import {alice, bob} from '../../wallet/__test__/fixtures/participants';
-import {SignState} from '../actions';
+import {testKnex as knex} from '../../../jest/knex-setup-teardown';
+import {MockChainService} from '../../chain-service';
+import {defaultTestConfig} from '../../config';
+import {createLogger} from '../../logger';
 import {ChainServiceRequest, requestTimeout} from '../../models/chain-service-request';
-import {channel} from '../../models/__test__/fixtures/channel';
+import {DBCloseChannelObjective} from '../../models/objective';
+import {Store} from '../../wallet/store';
+import {WalletResponse} from '../../wallet/wallet-response';
+import {TestChannel} from '../../wallet/__test__/fixtures/test-channel';
+import {ChannelCloser} from '../close-channel';
 
-import {applicationProtocolState} from './fixtures/protocol-state';
+const FINAL = 10; // this will be A's state to sign
+const testChan = TestChannel.create({aBal: 5, bBal: 5, finalFrom: FINAL});
```

Comment:
```suggestion
const finalFrom = 10; // this will be A's state to sign
const testChan = TestChannel.create({aBal: 5, bBal: 5, finalFrom});
```
and so on